src/SourceLocator/SourceLocator.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Asgrim\SourceLocator;
+
+interface SourceLocator
```

Comment:
@asgrim still states `$className` there.

What if I want to locate from a function name?